src/main.rs

```diff
@@ -182,6 +183,15 @@ struct Args {
         hide = true
     )]
     max_manifest_bucket_bytes: i64,
+    /// The storage type of nats streams, maps to [`StorageType`](async_nats::jetstream::stream::StorageType)
+    /// Possible values: "File" or "Memory", default is `File`
+    #[arg(
+        long = "stream-persistence",
+        env = "WADM_STREAM_PERSISTENCE",
+        default_value_t = StreamPersistence::File,
```

Comment:
why use `nats::StreamPersistence` as a proxy instead of directly using the mentioned type `[`StorageType`](async_nats::jetstream::stream::StorageType`?